SetupDataPkg/Tools/VariableList.py

```diff
@@ -252,6 +252,7 @@ def __init__(self, xml_node):
 
         self.help = xml_node.getAttribute("help")
         self.values = []
+        self.default = 0
```

Comment:
This does get set by the xml default attribute when the xml is loaded. This was failing because the xml loading code was trying to assign a default value and self.default did not exist.